jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -146,6 +146,17 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
              readOnly: true,
            },
          ] else []) +
+        (if api.config.tenants.tenants != {} then [
```

Comment:
`api.config.tenants.tenants` will allways be an array if it exists. So I think it's better to check `if std.objectHas(api.config.tenants, 'tenants')`

Reply:
done